roles/secretary.md

```diff
@@ -22,16 +22,14 @@ The primary responsibilities of the Discussion Session Coordinators are:
 
 ## Resources and Checklists
 
-Attendance spreadsheet is here: [link forthcoming]()
-
 Board repository is here: [https://github.com/swcarpentry/board/](https://github.com/swcarpentry/board/)
 
-Mentoring committee materials are here [https://github.com/swcarpentry/board/tree/master/subcommittees/mentoring](https://github.com/swcarpentry/board/tree/master/subcommittees/mentoring)
+Mentoring committee documents are located in [Carpentries organisation](https://github.com/carpentries) in its own [repository](/)
```

Comment:
This is nit-picky, but, this ordering sounds better to me: 
Mentoring committee documents are located in their own [repository](/) within the [Carpentries organisation](https://github.com/carpentries).

Reply:
well, it is :)
fixed in 92d0d8da92d4d6dd37007390c5bc2bfab8905d8f